staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -318,3 +319,7 @@ func parseTimeout(str string) time.Duration {
 	}
 	return 30 * time.Second
 }
+
```

Comment:
It might be a good idea to add a TODO here explaining that this will be moved to our versioned structs like other parameters before being implemented for real.